### DISCUSSION TITLE
How to revert ？？

### DISCUSSION THREAD

**Discussion by TomTom18-eng** [community/community#37640] [Other Feature Feedback, Questions, & Ideas]
How to revert to the previous version, I can't see the messages I follow now! !
![image](https://user-images.githubusercontent.com/50043324/198647031-5f80bba4-ef5c-4655-b1e1-97ff7eb678c1.png)

**Comment by SilasRodrigues19** [comment]
Hi @TomTom18-eng 

Does this option below not appear for you?

![image](https://user-images.githubusercontent.com/47532703/198653583-dbdbc851-42ec-460c-9873-0913c2375904.png)

**Reply by jevinskie** [reply]
No, we are stuck in A/B Hell where the option has been removed. I follow 3.8k people and have nearly 6k starred repos but have gotten less than a dozen items per day in the new, forced Feed with no option to go back to the original style. It makes GitHub no better than bitbucket or gitlab.

**Reply by SilasRodrigues19** [reply]
It's definitely a bug, in my case the "Following" button appears and in your case it doesn't
